app/containers/MessageBox/index.tsx

```diff
@@ -76,7 +76,7 @@ const libraryPickerConfig: Options = {
 	multiple: true,
 	compressVideoPreset: 'Passthrough',
 	mediaType: 'any',
-	forceJpg: true
+	forceJpg: false
```

Comment:
This way you would fix the issue you're trying to fix, but stop converting images to jpg.
It would be a regression.

Reply:
`forceJpg` flag set to `false` to stop converting images to `.jpg`, previously it was set to `true` that's why `.gif` images was converted to `.jpg` and was displayed as static images.

Reply:
It doesn't change the fact that we would still have issues with `.HEIC` rendering on the web if we don't convert to `.jpg`.
That's why I said it's a regression.

Reply:
![image](https://user-images.githubusercontent.com/1501599/201935535-5338b9b0-7b3d-4d06-a2b8-1b4e7b411782.png)

HEIC image displayed on web correctly, is there any other issue, please let me know. Please check attach image
Also if we just change extension of HEIC image would it work as expected on web?
@diegolmello

Reply:
Ok. Thanks, we're discussing this internally.
We have a few issues rendering HEIC depending on the OS/browser and the client how uploaded it.